lib/contentprovider/xbmcprovider.py

```diff
@@ -176,6 +176,15 @@ def play(self,item):
             il = self._extract_infolabels(item['info'])
             if len(il) > 0: #only set when something was extracted
                 li.setInfo('video',il)
+            # new art handling from Jarvis - begin
+            if 'art' in item['info'].keys():
+                if 'poster' in item['info']['art'].keys():
+                    # movie
+                    li.setArt({'thumb' :item['info']['art']['poster']})
+                elif 'thumb' in item['info']['art'].keys():
+                    # tv serie
```

Comment:
AFAIK, thumbnail je nahlad. Z tvojich komentarov to ale vyzera tak, ze `poster` sa pouziva len pre filmy a `thumb` iba pre serialy.